doc/telescope.txt

```diff
@@ -23,6 +23,25 @@ telescope.setup({opts})                                    *telescope.setup()*
     border: ~
         Boolean defining if borders are added to Telescope windows.
 
+        Default: true
+
+                                           *telescope.defaults.cache_limit*
+    cache_limit: ~
+        Will restrict the number of cached entries.
+
+        Only relevant if `cache_picker = true`. Set to -1 to avoid
+        caching limit.
+
+        Default: 10000
```

Comment:
I think our default could be even smaller if it affects performance at all.

Reply:
Agreed